docs/13-System-Architecture-Document.md

```
@@ -0,0 +1,1199 @@
+    API -- "SQL via ORM" --> DB
+    API -- "Read/Write" --> STG
+    API -- "HTTPS" --> OAI
+    API --> LOGS
+    SEED --> DB
+    SEED --> STG
+```
+
+| Ambiente | Propósito | Componentes | Datos | Notas |
+|----------|-----------|-------------|-------|-------|
+| Local development | Desarrollo y testing local | Frontend dev server + Backend dev + PostgreSQL local (o Docker) + MockAIProvider | Seed local; storage local | Captcha mockeado; `LLMProvider=mock` por defecto |
+| QA / demo | Verificación funcional y rehearsal de demo | Frontend desplegado + Backend desplegado + PostgreSQL gestionada o local + `OpenAIProvider` opcional | Seed completo `/docs/11` | Toggle `LLMProvider=openai|mock` por env |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 5; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)